src/app/job-routing/my-application/my-application.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="container" *ngFor="let myApplication of Applications">
-    <a href="#">
```

Comment:
why <a> tag again

Reply:
on clicking the div, it takes you to a different page that shows the full description of the job